### PR TITLE
Provision to add custom rq queues

### PR DESCRIPTION
**Pull request by sharathc3**
Pull-Request

- [x] Have you followed the guidelines in our Contributing document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
- [x] Have you lint your code locally prior to submission?
- [x] Have you successfully run tests with your changes locally?
- [x] Does your commit message have an explanation for your changes and why you'd like us to include them?
- [ ] Docs have been added / updated
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Did you modify the existing test cases? If yes, why?

---

What type of a PR is this? 

- [ ] Changes to Existing Features
- [ ] New Feature Submissions
- [ ] Bug Fix
- [x] Breaking Change
Default Queue definition in background jobs is moved to commom site config.
Without the migration, the existing setups will break.

--- 

- Motivation and Context (What existing problem does the pull request solve):
Queues in RQ was a hard coded into the system.
Now the hard coded values are removed and provision to add them via common_site config is added.

### CODE REVIEW

**Comment by codingCoffee** [comment]
also add the patch file path to https://github.com/frappe/bench/blob/master/bench/patches/patches.txt

**Comment by Sharath-Chikkanna** [comment]
patch file path added to patches.txt

File: bench/patches/v4/update_queues.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from bench.config.common_site_config import get_queues, get_config, update_config
+
+
+def execute(bench_path):
+    queues = get_queues()
+    config = get_config(bench_path)
+    for k1, v1 in queues.items():
+        for k2, v2 in v1.items():
+            queues[k1][k2]["workers"] = config["background_workers"]
```
Comment (sachinhub) [inline bench/patches/v4/update_queues.py]: Can you please provide sample config for 'background_workers' ?

**Comment by fproldan** [comment]
@Sharath-Chikkanna Hi, any chances to revive this?

**Comment by mhbu50** [comment]
@saurabh6790 Any plan to add this?

**Comment by fproldan** [comment]
+1

**Comment by barredterra** [comment]
Closing this because it's been inactive for over two years and still not ready to be merged. The developer doesn't seem to be active on GitHub anymore.

If you need this feature please open a new PR, add tests and docs and make sure that the tests succeed.

**Pull request closed without merging**